src/main/java/com/powsybl/network/hypothesis/HypothesisUtils.java

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.network.hypothesis;
+
+import com.powsybl.iidm.network.*;
+
+import java.util.function.BiConsumer;
+
+/**
+ * Some useful utility methods to create network hypotheses.
+ *
+ * @author Miora Ralambotiana <miora.ralambotiana at rte-france.com>
+ */
+public final class HypothesisUtils {
+
+    /**
+     * Split a given line and create a fictitious voltage level at the junction.<br>
+     * The line is considered split in half such as this:<br>
+     * <code>r1 = 0.5 * r</code><br>
+     * <code>r2 = 0.5 * r</code><br>
+     * If the line is between two BUS-BREAKER voltage levels, the fictitious voltage level will be BUS-BREAKER and the two new lines
+     * will be linked to a fictitious bus in this voltage level.<br>
+     * If the line is between the two NODE-BREAKER voltage levels, the fictitious voltage level will be NODE_BREAKER and the
+     * two lines will be linked via an internal connection between two nodes in this voltage level.
+     */
+    public static void createVoltageLevelOnLine(Line line) {
+        createVoltageLevelOnLine(0.5, 0.5, 0.5, 0.5, 0.5, 0.5, line);
+    }
+
+    /**
+     * Split a given line and create a fictitious voltage level at the junction.<br>
+     * The characteristics of the two new lines respect the given ratios such as this:<br>
+     * <code>r1 = rdp * r</code><br>
+     * <code>r2 = (1 - rdp) * r</code><br>
+     * If the line is between two BUS-BREAKER voltage levels, the fictitious voltage level will be BUS-BREAKER and the two new lines
+     * will be linked to a fictitious bus in this voltage level.<br>
+     * If the line is between the two NODE-BREAKER voltage levels, the fictitious voltage level will be NODE_BREAKER and the
+     * two lines will be linked via an internal connection between two nodes in this voltage level.
+     */
+    public static void createVoltageLevelOnLine(double rdp, double xdp, double g1dp, double b1dp, double g2dp, double b2dp,
```

Comment:
Maybe it could be easy here to add a boolean `breakers`. If true, you have to put two breakers and if not, internal connections or nothing. What do you think ?

Reply:
If true, you are think of doing:
`` line1 ------- node1 -----(sw1) ---- node2 -----(sw2)------ node3 ------- line2``
or
`` line1 ------- node1 ------(sw)------ node2 -------line2``
?

Reply:
Where is the new voltage level ? In node 2 ? So it is the first proposition for me, with a new equipment that will be connected to node2. @flo-dup what do you think ? node 2 is surrounded by breakers or by internal connections.

Reply:
I'd vote for the first proposition, as it's the common topology for voltage levels. But I think you're mistaken when asking where the new voltage level is. The new voltage level is all the nodes and switches together: for instance in first proposition the voltage level is the group of nodes node1, node2, node3, together with switches sw1 and sw2.

Reply:
Indeed I have written node but I mean bus = node 1 + node 2 + node 3 + swicthes and the new equpiment.